src/configs/routes.js

```diff
@@ -25,6 +25,7 @@ const routes = {
   pathToAddContact: '/contacts/add',
   pathToOrders: '/orders',
   pathToComments: '/comments',
+  pathToSizes: '/sizes',
   pathToOrderItem: '/orders/:id',
```

Comment:
add path for size editing by id like `/sizes/:id`
and for adding

Reply:
add paths